app/views/organisations/new.html.erb

```diff
@@ -6,16 +6,35 @@
 <%= form_for @organisation, url: organisations_path do |form| %>
   <div class="govuk-form-group <%= field_error(@organisation, "name") %>">
     <%= form.label :name, "Organisation name", class: "govuk-label" %>
-    <div class="govuk-hint">If your organisation name does not appear below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
+    <div class="govuk-hint">If your organisation name does not appear
+      below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
     <%= form.select :name, options_for_select(@register_organisations << "", ""), {}, class: "govuk-select" %>
   </div>
 
   <div class="govuk-form-group <%= field_error(@organisation, "service_email") %>">
-    <%= form.label :service_email, "Service email", class: "govuk-label" %>
+    <% if @organisation.errors.attribute_names.include?(:name) %>
+      <span class="govuk-error-message" id="location_error">
```

Comment:
Something not quite right here? Looks like it's checking for errors on `name` but in the `service_email` field?

Also we have an error with a link in which is breaking things... this may be from before this change, but I think we need to remove the link. 

<img width="751" alt="Screenshot 2021-09-07 at 06 09 13" src="https://user-images.githubusercontent.com/825374/132328227-3ff77a62-9e7b-4dde-aa29-91994da09e05.png">

Reply:
Label removed and link removed from YAML.